tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
```

Comment:
I'm not sure what am I doing wrong but I'm trying to test this manually on kcli, using slightly modified version that deploys the kbs. When this line (`load confidential_kbs.sh`) is present I always get a skip:

```
bats --show-output-of-passing-tests --verbose-run k8s-confidential-attestation.bats
k8s-confidential-attestation.bats

load
load
   Test we can get evidence from CDH resource                                                                      1/1
   bats warning: Executed 0 instead of expected 1 tests
```

When I remove it it executes the setup (where I manually added skip to test it):

bats --show-output-of-passing-tests --verbose-run k8s-confidential-attestation.bats
k8s-confidential-attestation.bats

```
load
load
 - Test we can get evidence from CDH resource (skipped: FOO)
   setup
   teardown
   error: arguments in resource/name form must have a single resource and name
   error: the path "/pod-attestable.yaml" does not exist
```

Is it supposed to work there? Would you happen to know what am I missing (a variable, perhaps?)?

Reply:
@ldoktor the problem is that the `confidential_kbs.sh` does not exists currently in the `main` branch this is being introduced on https://github.com/kata-containers/kata-containers/pull/9065/files which needs to be merge first and later this PR

Reply:
currently this PR depends on several PRs to be merged first

Reply:
That was not the case, I was using the Wainer's PR and your test on top. I was wondering where the skip comes from as I couldn't locate that. It's not that important though, I was just wondering whether you could see some obvious reason from top of your head (like it requires AKS, TEE or whatever)

Reply:
hi @ldoktor ! Try to source `tests_common.sh` after `confidential_kbs.sh`.  Because the later will source some files too, I suspect it is getting `$pod_config_dir` (which is set by the former tests_common.sh) overwrite.